config/operator/operator/kustomization.yaml

```diff
@@ -5,4 +5,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: k3d-kyma-registry:5001/serverless-operator-dev-local
-  newTag: 1.1.0
+  newTag: 1.0.0
```

Comment:
Why in this place we have version `1.0.0`?
Is this important or it's value for override?

Reply:
it's value for override but should be `0.0.0` I'll fix it